e2e/_suites/fleet/services.go

```diff
@@ -163,7 +163,7 @@ func runElasticAgentCommand(profile string, image string, service string, proces
 // be defined by that value
 // Else, if the environment variable BEATS_USE_CI_SNAPSHOTS is set, then the artifact
 // to be downloaded will be defined by the latest snapshot produced by the Beats CI.
-func downloadAgentBinary(artifact string, version string, OS string, arch string, extension string) (string, string, error) {
+func downloadAgentBinary(artifact string, version string, OS string, arch string, extension string, stale bool) (string, string, error) {
```

Comment:
As an improvement in a follow-up PR, I'm interested in creating a BinaryRequest struct to hold the references needed to download the binary, which would simplify the signature of the method.